src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1084,39 +1084,30 @@ internal int GetPortModelIndex(PortModel portModel)
         }
```

Comment:
This code was simply updated to honor `PortModel.LineIndex` if it is specified, or `PortModel.Index` otherwise. No more differentiation between in/out ports.